cli/clitest/clitest.go

```diff
@@ -168,6 +168,11 @@ func StartWithAssert(t *testing.T, inv *serpent.Invocation, assertCallback func(
 		switch {
 		case errors.Is(err, context.Canceled):
 			return
+		case err != nil && strings.Contains(err.Error(), "driver: bad connection"):
+			// When a test exits it cancels the context. If a transaction is open
+			// and a query is being executed, instead of a context.Canceled error
+			// we get a "driver: bad connection" error.
+			return
```

Comment:
Wouldn't exiting with an open transaction be bugged behaviour? Could this mask someone forgetting to close a connection?

Reply:
seems to also be reported here: https://github.com/lib/pq/issues/1137